zcash_client_backend/src/data_api/error.rs

```diff
@@ -34,6 +35,12 @@ pub enum Error<NoteId> {
     /// Chain validation detected an error in the block at the specified block height.
     InvalidChain(BlockHeight, ChainInvalid),
 
+    /// A provided extsk is not associated with the specified account.
```

Comment:
This comment doesn't make sense (copy-paste error?) - the error kind is AFAICT for when a given account ID is not present in the wallet.